resources/views/admin.blade.php

```diff
@@ -109,28 +103,48 @@
                 </div>
               </div>
 
-            <table id="myTable" class="table table-striped table-dark table-hover  " style="width: 100%"></table>
+            <table id="myTable" class="table table-striped table-dark table-hover " style="width: 100%" data-turbolinks="false"></table>
```

Comment:
explain why you have used `turbolinks` and make the row size smaller

![image](https://github.com/user-attachments/assets/266f9632-1f47-41ea-bfd7-627402538b56)

Reply:
i tried to edit the datatable layout styles by manipulating the bootstrap applied to it....that time i added this after referring to a post...cuz bootstrap wasn't new style weren't applying.....ig i forgot to remove it after i figured to changed the layout using data table..Ill remove it @Raza9798